vm/vm.go

```diff
@@ -433,6 +433,34 @@ func (vm *VM) Initialize(
 	webSocketServer, pubsubServer := rpc.NewWebSocketServer(vm, vm.config.StreamingBacklogSize)
 	vm.webSocketServer = webSocketServer
 	vm.handlers[rpc.WebSocketEndpoint] = pubsubServer
+
+	has, err = vm.HasLastProcessed()
+	if err != nil {
+		snowCtx.Log.Error("could not determine if have last processed")
+		return err
+	}
+	if has {
+		lastProcessedHeight, err := vm.GetLastProcessedHeight()
+		if err != nil {
+			snowCtx.Log.Error("could not get last processed height", zap.Error(err))
+			return err
+		}
+
+		for height := lastProcessedHeight; height <= vm.lastAccepted.Height(); height++ {
+			var blk *chain.StatelessBlock
+			if height == vm.lastAccepted.Height() {
+				blk = vm.lastAccepted
+			} else {
+				blk, err = vm.GetDiskBlock(ctx, height)
+				if err != nil {
+					vm.snowCtx.Log.Info("could not find block on-disk", zap.Uint64("height", height))
+					return err
+				}
+			}
+			vm.processAcceptedBlock(blk)
+		}
+	}
```

Comment:
Instead of processing them synchronously within `Initialize`, could we add them to the accepted queue?

We should add them to the queue synchronously to guarantee that accepted blocks are processed by any listeners in order.

Reply:
Done, the blocks are now added to the queue so that they are not blocking the initialization of the `VM` with `vm.acceptedQueue <- blk`